docs/blog/newsletter-july-2019.rst

```diff
@@ -0,0 +1,132 @@
+
+Forcing a hard wrap at a particular character count or column helps avoid `reflow problems <https://www.w3.org/TR/WCAG21/#reflow>`_, but causes other problems. For example, hard wraps:
+
+* create awkward line breaks on devices with different screen sizes
+* wreak havoc on diffs
+* make text more difficult to parse for people who are translating to different languages and people with cognitive disabilities
+
+Hard wraps also interfere with using filters like `grep <http://man7.org/linux/man-pages/man1/grep.1.html>`_ to search for words and phrases within a file. Soft wrap is an option for preserving searchability, but it requires everyone to use a text editor with soft wrap support.
+
+Several people mentioned `one sentence per line <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_ as a solution, noting that it's the `recommended approach for AsciiDoc <https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/asciidoc-recommended-practices.adoc#one-sentence-per-line>`_.
+
+-----------------------------------------------------------------
+From our sponsor: Spotlight on The Not-Boring Tech Writer podcast 
+-----------------------------------------------------------------
+
+This month's newsletter is sponsored by KnowledgeOwl!
+If you're interested in sponsoring our newsletter,
+the is more information in our `sponsorship prospectus </newsletter/sponsorship/`.
+
+.. raw:: html
+
+    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
+      <tbody>
+        <tr>
+          <td width="75%">
+              <p>
+              Do you love documentation and podcasts? Check out our <a class="reference external" href="https://www.knowledgeowl.com/home/not-boring-tech-writer-podcast-relaunch">spotlight on The Not-Boring Tech Writer podcast</a>. Each episode focuses on a single skill or tool to provide you with actionable content.
+              </p>
+              <p>
+              KnowledgeOwl is proud to sponsor TNBT podcast along with the WTD newsletter. KnowledgeOwl makes knowledge base software and loves to help documentarians. Check out the <a class="reference external" href="https://www.knowledgeowl.com/home">KnowledgeOwl website</a> to learn more.
+              </p>
+          </td>
+          <td width="25%">
+              <img alt="knowledgeowl" src="../../_images/knowledge-owl-logo.png">
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    
+---------
+Job posts
+---------
+
+`Knowledge Management Strategist <https://jobs.writethedocs.org/job/127/knowledge-management-strategist/>`__
+
+`Project-Based Technical Writer <https://jobs.writethedocs.org/job/125/project-based-technical-writer/>`__
```

Comment:
This job isn't current.

Reply:
Oh! I swear it was there yesterday. Thanks for spotting, I'll remove